modules/running-network-verification-manually-cli.adoc

```diff
@@ -19,18 +19,42 @@ When you run the network verification, you can specify a set of VPC subnet IDs o
 
 .Procedure
 
-* Verify the network configuration by using one of the following methods: 
+* Verify the network configuration by using one of the following methods:
 ** Verify the network configuration by specifying the cluster name. The subnet IDs are automatically detected:
 +
 [source,terminal]
 ----
-$ rosa verify network -c <cluster_name> <1>
+$ rosa verify network --cluster <cluster_name> <1>
 ----
 <1> Replace `<cluster_name>` with the name of your cluster.
 +
 .Example output
 [source,terminal]
 ----
+I: Verifying the following subnet IDs are configured correctly: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
+I: subnet-0b761d44d3d9a4663: pending
+I: subnet-0f87f640e56934cbc: passed
+----
+*** Ensure that verification to all subnets has been completed.
++
+[source,terminal]
+----
+$ rosa verify network --watch <1>
+                      --status-only <2>
+                      --region us-east-1 <3>
+                      --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc <4>
```

Comment:
I copy the command but it is not valid.
```
rosa verify network --watch 
Error: unknown flag: --watch
Usage:
  rosa verify [command]

Available Commands:
  openshift-client Verify OpenShift client tools
  permissions      Verify AWS permissions are ok for non-STS cluster install
  quota            Verify AWS quota is ok for cluster install

Flags:
  -h, --help   help for verify

Global Flags:
      --color string   Surround certain characters with escape sequences to display them in color on the terminal. Allowed options are [auto never always] (default "auto")
      --debug          Enable debug mode.

Use "rosa verify [command] --help" for more information about a command.

Failed to execute root command: unknown flag: --watch
[yingzhan@localhost rosa]$                       --status-only 
bash: --status-only: command not found...
[yingzhan@localhost rosa]$                       --region us-east-1 
bash: --region: command not found...
[yingzhan@localhost rosa]$                       --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc 
bash: --subnet-ids: command not found...

```

Reply:
Did you copy the entire command? 
`$ rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc`.

This is an example output from the SME:

```[user]$ rosa verify network --cluster xxxx
I: Verifying the following subnet IDs are configured correctly: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
I: subnet-0b761d44d3d9a4663: pending
I: subnet-0f87f640e56934cbc: passed
I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc
[user]$ rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc
I: Checking the status of the following subnet IDs: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
I: subnet-0f87f640e56934cbc: passed
I: subnet-0b761d44d3d9a4663: passed
[user]$ rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc
I: Checking the status of the following subnet IDs: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
I: subnet-0b761d44d3d9a4663: passed
I: subnet-0f87f640e56934cbc: passed
[thomasmckay@thomasmckay rosa{master}]$ rosa verify network  --status-only --region us-east-1 --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc
I: Checking the status of the following subnet IDs: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
I: subnet-0b761d44d3d9a4663: passed
I: subnet-0f87f640e56934cbc: passed
[user]$ rosa verify network  --status-only --cluster tomckay
I: Checking the status of the following subnet IDs: [subnet-0b761d44d3d9a4663 subnet-0f87f640e56934cbc]
I: subnet-0b761d44d3d9a4663: passed
I: subnet-0f87f640e56934cbc: passed

```

Reply:
I click the back Copy icon and then paste it to run.  I paste the content here
```
rosa verify network --watch 
                      --status-only 
                      --region us-east-1 
                      --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc 
```

Reply:
Hi @thomasmckay. Can you please advise on this? Thanks.

Reply:
I'm not sure I understand the question. The docs don't have line continuation characters (`\`) but instead have numbers for documentation (`<1>`).

Reply:
Thanks @thomasmckay. I added the `\` as they should have been there but @yingzhanredhat says the command does not work.

Reply:
Hi @yingzhanredhat, I ran the following commands and they worked for me: 
```
$ rosa verify network --cluster aecollin
I: Verifying the following subnet IDs are configured correctly: [subnet-02b865f670e8283ef subnet-0d33a8516675d1a4c]
I: subnet-02b865f670e8283ef: pending
I: subnet-0d33a8516675d1a4c: passed
I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --subnet-ids subnet-02b865f670e8283ef,subnet-0d33a8516675d1a4c
```  
```
$ rosa verify network --watch --status-only --region us-east-1 --subnet-ids subnet-02b865f670e8283ef,subnet-0d33a8516675d1a4c
I: Checking the status of the following subnet IDs: [subnet-02b865f670e8283ef subnet-0d33a8516675d1a4c]
I: subnet-0d33a8516675d1a4c: passed
I: subnet-02b865f670e8283ef: passed
```

Reply:
@yingzhanredhat Can you please review the amended command? Many thanks.